src/providers/sh/commands/scale.js

```diff
@@ -244,6 +250,12 @@ async function run({ token, sh: { currentTeam } }) {
     return
   }
 
+  // Don't scale down the deployment if multiple aliases are linked to the same deploymentId
+  if (match.scale.current > scaleArg && duplicateDeployments.length > 1) {
+    console.log(`Scaling cannot be performed as there are multiple aliases linked to ${match.url}`)
+    return process.exit(0)
```

Comment:
I wonder what the correct behavior should actually be:
probably we want to allow scaling up the new deployment, but not scale down the old one.

So with this approach, we would not even allow scaling up on alias if multiple aliases are pointing to the old URL.
So maybe this check should be just after scaling up the new instance, and then instead of scaling down the old one we would log this and proceed normally (or do we have a helper for that...)